From ab6be41ea043065424cb494ec80d4242812e2488 Mon Sep 17 00:00:00 2001
From: Robert Bragg <robert@linux.intel.com>
Date: Mon, 04 Jul 2011 14:12:20 +0000
Subject: build: update cogl_onscreen_x11 syms to cogl_x11_onscreen

Cogl recently renamed symbols with the form
cogl_onscreen_<platform>_blah to be consistent with other platform
specific APIs so they are now named like cogl_<platform>_onscreen_blah.
This makes the corresponding change to clutter.
---
diff --git a/clutter/cogl/clutter-stage-cogl.c b/clutter/cogl/clutter-stage-cogl.c
index a39c03e..111605e 100644
--- a/clutter/cogl/clutter-stage-cogl.c
+++ b/clutter/cogl/clutter-stage-cogl.c
@@ -124,7 +124,7 @@ clutter_stage_cogl_realize (ClutterStageWindow *stage_window)
 #ifdef COGL_HAS_XLIB_SUPPORT
   if (stage_x11->xwin != None)
     {
-      cogl_onscreen_x11_set_foreign_window_xid (stage_cogl->onscreen,
+      cogl_x11_onscreen_set_foreign_window_xid (stage_cogl->onscreen,
                                                 stage_x11->xwin,
                                                 _clutter_stage_x11_update_foreign_event_mask,
                                                 stage_x11);
@@ -160,7 +160,7 @@ clutter_stage_cogl_realize (ClutterStageWindow *stage_window)
 
 #ifdef COGL_HAS_XLIB_SUPPORT
   if (stage_x11->xwin == None)
-    stage_x11->xwin = cogl_onscreen_x11_get_window_xid (stage_cogl->onscreen);
+    stage_x11->xwin = cogl_x11_onscreen_get_window_xid (stage_cogl->onscreen);
 
   return clutter_stage_window_parent_iface->realize (stage_window);
 #else
--
cgit v0.9
